### ISSUE TITLE
add configurable configuration location

### ISSUE THREAD

**Issue by evanrich** [kpetremann/salt-exporter#63]
excellent tool, one quick suggestion:  make the configuration file location a parameter, something like `salt-exporter --config /etc/salt-exporter.yaml`

I'm placing the salt-exporter and salt-live binaries in /usr/local/bin, but as of right now I see no way to keep the configuration  file in a better path.

**Comment by evanrich** [comment]
edit: it _seems_ like if i run `./salt-exporter /home/config.yaml` it works maybe?   IF so, can this be documented in the `salt-exporter --help` list?

**Comment by kpetremann** [comment]
hello @evanrich,

good point, I am on it.

> excellent tool
thanks :)

**Comment by kpetremann** [comment]
implemented in the [v0.10.0](https://github.com/kpetremann/salt-exporter/releases/tag/v0.10.0) release.

```
salt-exporter --config-file="/srv/salt-exporter/config.yml"
```
